shared/studio/components/explainVis/state.ts

```diff
@@ -51,23 +52,13 @@ export class ExplainState extends Model({
   parentCtxId: prop<number | null>(null).withSetter(),
   hoveredCtxId: prop<number | null>(null).withSetter(),
```

Comment:
These three `...ctxId` props aren't needed I think? They should be derivable from the `selectedPlan` and `hoveredPlan` states.